### DISCUSSION TITLE
Using the glossary `term` role in code outputs

### DISCUSSION THREAD

**Discussion by tsalo** [executablebooks/meta#713] [Q&A]
I am trying to reproduce a `mkdocs` site with Jupyter Book, and one thing I am stuck on is linking to the glossary within code outputs. The site includes many tables and text sections that are generated with Python functions and converted to Markdown.
I initially had trouble making code-generated  Markdown tables render correctly, but I got around that with the following:

```
from IPython.display import display, HTML
from markdown import markdown

# table is a pandas DataFrame
display(HTML(markdown(table.to_markdown(), extensions=['tables'])))
```

The table looks good, but terms within the table end up being converted to HTML.

For example, `` {term}`Manufacturer (metadata)` `` becomes: 
![image](https://user-images.githubusercontent.com/8228902/162253498-11b345bf-06e6-4e19-bc8a-c579f757fb3d.png)

I got around this by writing a function that replaces the `<code>` tag with backticks in the HTML string, before it's displayed. 
Now, I get the literal string, rather than the linked glossary term:

![image](https://user-images.githubusercontent.com/8228902/162253751-62cc5bd2-b746-4078-baf9-5f1e6290b6b8.png)

The repository I am working on is [`bids-standard/bids-specification-ignore`](https://github.com/bids-standard/bids-specification-ignore).

TLDR: Is there any way to include terms that will link to the glossary in code outputs (especially Markdown strings and tables)? Is there a more straightforward solution that I am missing?
